### ISSUE TITLE
Compile with HybridCRT

### ISSUE THREAD

**Issue by apkipa** [BreeceW/WinUIEdit#12]
Using [Hybrid CRT](https://github.com/microsoft/terminal/blob/8598ed78d58cf6b10372fca4132bfa9553ac6735/src/common.build.post.props#L14) would allow this awesome control to be used both in UWP and XAML Islands v1, without the need of VCRTForwarders.

**Comment by BreeceW** [comment]
I agree. Now that WinUI 2.8 is compiled with Hybrid CRT, I will follow for WinUIEdit.Uwp. WinUIEdit (for WinUI 3) is already compiled with Hybrid CRT. I will enable the same on UWP in the next update.

**Comment by BreeceW** [comment]
WinUIEdit.Uwp 0.0.4-prerelease is published on NuGet with Hybrid CRT.
